src/cmd/oui.rs

```diff
@@ -186,3 +339,43 @@ where
         .collect();
     results
 }
+
+fn print_update_txn(
+    txn: &BlockchainTxnRoutingV1,
+    envelope: &BlockchainTxn,
+    status: &Option<PendingTxnStatus>,
+    format: OutputFormat,
+) -> Result {
+    let update = match txn.update.as_ref().unwrap() {
+        UpdateTxn::UpdateRouters(txn) => {
+            let mut str = String::from("Routing ");
+            let addr = map_addresses(txn.router_addresses.clone(), |v| v.to_string())?;
+            str.extend(addr);
+            str
+        }
+        UpdateTxn::NewXor(_) => "NewXor".into(),
+        UpdateTxn::UpdateXor(txn) => format!("Update Xor {}", txn.index),
+        UpdateTxn::RequestSubnet(size) => format!("Request subnet of size {}", size),
+    };
+
+    match format {
+        OutputFormat::Table => {
+            ptable!(
+                ["Key", "Value"],
+                ["OUI", txn.oui],
+                ["Update", update],
+                ["Hash", status_str(status)]
+            );
+            print_footer(status)
+        }
+        OutputFormat::Json => {
+            let table = json!({
+                "OUI": txn.oui + 1,
```

Comment:
Why are you lying both the oui here? lets just reflect what it actually is

Reply:
This was shifted before but perhaps it was fixed.